cloudsql/cloudsql.go

```diff
@@ -32,3 +32,11 @@ type Dialer interface {
 
 	io.Closer
 }
+
+// Logger is the interface used throughout the project for logging.
+type Logger interface {
```

Comment:
This looks like a public interface now right? Does this mean it's in line for a breaking change?

Reply:
Yes, it is public. I've added `Debugf` to the interface to support https://github.com/GoogleCloudPlatform/cloudsql-proxy/issues/1255. Since this interface is meant for google-internal usage only, I think we could technically break it by adding additional methods if necessary. But between info, error, and debug, we have everything we need now.